src/FunctionApps/python/IntakePipeline/fhir.py

```diff
@@ -37,4 +42,158 @@ def read_fhir_bundles(container_url: str, container_prefix: str) -> Iterator[dic
 
 
 def upload_bundle_to_fhir_server(bundle: dict) -> None:
-    pass
+    """Upload a FHIR bundle to FHIR server"""
+    process_fhir_resource(bundle)
+
+
+def process_fhir_resource(
+    fhir_json: dict = {}, fhir_string: str = "", fhir_filepath: str = ""
+):
+    """Process a file containing FHIR resource(s) enclosed by ndjson
+    :param str filename: FHIR resource(s) enclosed by ndjson
+    """
+    if len(fhir_json) > 0:
+        pass
+    elif os.path.exists(fhir_filepath):
+        fhir_string = open(fhir_filepath).read()
+        fhir_json = json.loads(fhir_string)
+    else:
+        fhir_json = json.loads(fhir_string)
+
+    import_to_fhir(fhir_json)
+
+
+def import_to_fhir(fhir_json: dict, method: str = "PUT"):
+    """Import a FHIR resource to the FHIR server.
+    The submissions may Bundles or individual FHIR resources.
+
+    See :py:func:`_ensure_bundle_batch` for details about Bundle
+    conversion and handling.
+
+    :param dict fhir_json: FHIR resource in json format.
+    :param str method: HTTP method to use (currently PUT or POST supported)
+    """
+    try:
+        token = get_access_token()
+    except Exception:
+        logging.exception("Failed to get access token")
+        raise requests.exceptions.HTTPError(
+            "Authorization error occurred while processing information into \
+            FHIR server."
+        )
+    retry_strategy = Retry(
+        total=3,
+        status_forcelist=[429, 500, 502, 503, 504],
+        allowed_methods=["HEAD", "PUT", "POST", "OPTIONS"],
+    )
+    adapter = HTTPAdapter(max_retries=retry_strategy)
+    http = requests.Session()
+    http.mount("https://", adapter)
+    http.mount("http://", adapter)
+    fhir_url = os.environ.get("FHIR_URL", "")
```

Comment:
We're passing these things in from `__init__.py` - check the `get_required_config` function

Reply:
Done.